### PR TITLE
 #162163235 A user should be able to like and dislike articles.

### PR DESCRIPTION
**Pull request by KicaRonaldOkello**
#### What does this PR do?

- This PR ensures that a user can like or dislke an article.

- This PR also ensure that all readers can see the number of likes on an article.

#### Description of Task to be completed?

- Users can like articles.

- Users can also dislike articles.

-  Users not looged in are redirected to login before they can like or dislike an article.

#### Screenshots:
Below is the screenshot of an article returned with the _**like status**_ of the logged in user and the **_likes count_** for the article
image
Below is a screenshot when the user dislikes the article.
image
Below is a screenshot of a modal which pops up when an unlogged in user tries to like an article.
image

[Delivers #162163235]

### CODE REVIEW

**Review by Nta1e — APPROVED** [review]
LGTM

File: src/actions/LikesAction.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    .then(res => res.json())
+    .then(data =>
+      dispatch({
+        type: actionType.LOAD_LIKE_STATUS,
+        payload: data
+      })
+    );
+};
+
+const LikeIcon = slug => dispatch => {
```
Comment (JaymesKat) [inline src/actions/LikesAction.js]: `LikeStatus`, `LikeIcon` and `DisLikeIcon` are not meaningful names for actions. They should indicate an action being performed for instance `getLikesStatus`.

**Review by JaymesKat — CHANGES_REQUESTED** [review]
Update the PR heading to match the eng. playbook conventions

File: src/actions/LikesAction.js (hunk 1 above)
Comment (KicaRonaldOkello) [reply src/actions/LikesAction.js]: Okay, let me change that

**Review by SokoPaulSokool — APPROVED** [review]

**Review by bibangamba — APPROVED** [review]
Good job on fixing the state bug. The like button now reflects the actual status instead of holding reflecting some cached value.

**Review by JaymesKat — APPROVED** [review]
@KicaRonaldOkello Good work. Looks ready for merge.

**Pull request merged**